### PR TITLE
Update the use of Machines in Che

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
As discussed here https://github.com/eclipse/che/issues/13703  Machine terminology is deprecated and should be updated. This pr replaces usage of ```machine``` word with appropriate analog in some exceptions that might be exposed to end user.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13703

#### Release Notes
n/a

#### Docs PR
n/a

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
@eclipse/eclipse-che-qa can you take a look at test results?

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesBrokerInitContainerApplier.java [hunk 1]
```diff
@@ -77,7 +77,8 @@ public void apply(
           brokerEnvironment.getMachines().get(Names.machineName(brokerPod, container));
       if (brokerMachine == null) {
         throw new InfrastructureException(
-            String.format("Could not find machine for broker container %s", container.getName()));
+            String.format(
+                "Could not find appropriate place for broker container %s", container.getName()));
```
Comment (l0rd) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesBrokerInitContainerApplier.java]: Maybe "Could not retrieve the specification of the plugin broker container %s"?

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/hc/ServerChecker.java [hunk 2]
```diff
@@ -119,8 +119,7 @@ public void run() {
       if (isTimedOut()) {
         reportFuture.completeExceptionally(
             new InfrastructureException(
-                String.format(
-                    "Server '%s' in machine '%s' not available.", serverRef, machineName)));
+                String.format("Server '%s' in '%s' is not available.", serverRef, machineName)));
```
Comment (l0rd) [inline wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/hc/ServerChecker.java]: Why not replacing "machine" with "container"? This change makes it hard to understand the message.

**Comment by rhopp** [comment]
#13703 is set to milestone 7.1... Sooo -1 for merging?

**Comment by skabashnyuk** [comment]
@rhopp I've asked for clarification https://github.com/eclipse/che/issues/13703#issuecomment-510041985

**Comment by skabashnyuk** [comment]
@rhopp https://github.com/eclipse/che/issues/13703 is set to 7.0.0

**Comment by skabashnyuk** [comment]
@l0rd @rhopp @mshaposhnik can you approve this pr?

**Review by benoitf — APPROVED** [review]

**Review by l0rd — APPROVED** [review]

**Review by rhopp — APPROVED** [review]

**Pull request merged**